tests/suite/spec.rs

```diff
@@ -6889,8 +6889,7 @@ fn spec_test_511() {
 fn spec_test_512() {
     let original = r##"[link *foo **bar** `#`*](/uri)
 "##;
-    let expected =
-        r##"<p><a href="/uri">link <em>foo <strong>bar</strong> <code>#</code></em></a></p>
+    let expected = r##"<p><a href="/uri">link <em>foo <strong>bar</strong> <code>#</code></em></a></p>
```

Comment:
I wonder why several of these lines changed...are there different formatting rules now?

Reply:
It seems like it. I recently moved to the most recent Rust stable distribution. It's unfortunate, because these changes are basically noise. Is there a way of dealing with this?